README.md

```diff
@@ -157,7 +157,7 @@ The following needs to be done once at the beginning, and then later only when m
 #### Executing Build Script
 We use a custom build script that simplifies the process of building and overlaying catkin workspaces.
 
-To build our workspaces for ROS, in the `robosub-ros` directory, execute
+To build our workspaces for ROS, in the `/dev/robosub-ros` directory, execute
```

Comment:
This is technically incorrect. It should be either `~/dev/robosub-ros` or `/root/dev/robosub-ros`

Reply:
I made the change